### ISSUE TITLE
Can't use # key as keybind

### ISSUE THREAD

**Issue by shibii1111** [bluemagic123/tModLoader#2892]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

Can't use # on French Canada keyboard. It is the key on the left side of 1 aka OemTilde when I use English keyboard on windows. In Terraria 1.4 in my input profiles.json, it is known as OemQuotes, but for some reasons it won't let me use it at all in the controls menu, but I can write it in the chat. 

Works in vanilla Terraria 1.4 and was working on tmod 1.3

In the log bellow I'll attach the message it adds everytime I try to bind it. 

### Log File

[20:04:13.449] [Main Thread/WARN] [FNA]: KEY/SCANCODE MISSING FROM SDL2->XNA DICTIONARY: SDLK_HASH SDL_SCANCODE_GRAVE

### Steps to reproduce

1. Set French Canadian QWERTY keyboard in windows
2. open controls in terraria and try to use the key left to the "1" above "tab"

### Expected Behavior

Should be allowing me to bind it and seems to be knowns as OemQuotes in vanilla Terraria within the input profile json file.

### Actual Behavior

nothing happens and shows an error in the log file (client.log btw)

### Reproduction frequency

always.

### Additional Information

N/A

**Comment by DreiPixel** [comment]
i really can't believe that this isn't fixed for over 2 years. this bug is extremely annoying.
https://github.com/tModLoader/tModLoader/issues/1969 (same issue)

**Comment by JavidPack** [comment]
I took a look at this today, I might have a solution.

[ScriptCaller.zip](https://github.com/tModLoader/tModLoader/files/13142465/ScriptCaller.zip)

Take the file inside this zip and replace the existing file in the LaunchUtils folder within your install directory. Then launch the game. I tested it locally by changing the keyboard language, I'm not sure if a physical keyboard of a different language would behave differently. My testing suggests that it works the same as Terraria now, but it's hard to test since my keys don't match up. The keys are now detected at least, but they still will show OEM_Something that might not match up with what you see printed on your keyboard.

If anyone comes across this issue and can confirm the fix, we can look into making it the default.

**Comment by DreiPixel** [comment]
yeah that change works for me. 1 button is not working but everything else works. (the left/right arrow key left from Y is not working). i have a German QWERTZ Layout.
